view/theme/frio/js/dropzone-frio.js

```diff
@@ -0,0 +1,24 @@
+function isInteger(value) {
+	return /^\d+$/.test(value);
+}
+
+function getMBytes(value) {
```

Comment:
This works but I'd rather we didn't have the same method duplicated in both PHP and Javascript. Please perform the calculation in the PHP module like I suggested and remove this file.